sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/ActionDescriptorFactory.scala

```diff
@@ -189,26 +190,14 @@ private[impl] object ActionDescriptorFactory extends ComponentDescriptorFactory
 
     val serviceName = nameGenerator.getName(component.getSimpleName)
 
-    val serviceLevelOptions = {
-
-      val allOptions =
-        AclDescriptorFactory.serviceLevelAclAnnotation(component) ::
-        eventingInForEventSourcedEntityServiceLevel(component) ::
-        eventingInForValueEntityServiceLevel(component) ::
-        subscribeToEventStream(component) ::
-        publishToEventStream(component) :: Nil
-
-      val mergedOptions =
-        allOptions.flatten
-          .foldLeft(kalix.ServiceOptions.newBuilder()) { case (builder, serviceOptions) =>
-            builder.mergeFrom(serviceOptions)
-          }
-          .build()
-
-      // if builder produces the default one, we can returns a None
-      if (mergedOptions == kalix.ServiceOptions.getDefaultInstance) None
-      else Some(mergedOptions)
```

Comment:
We had this code copied in two places and I needed a third one, so moved it into `ComponentDescriptorFactory.mergeServiceOptions`